includes/storage/SQLStore/SMW_SQLStore3_Readers.php

```diff
@@ -387,17 +401,17 @@ protected function fetchSemanticData( $id, SMWDataItem $object = null, SMWSQLSto
 	 * property _SKEY. Only empty arrays will be returned there.
 	 *
 	 * @param SMWDIProperty $property
-	 * @param mixed $value SMWDataItem or null
-	 * @param SMWRequestOptions $requestoptions
+	 * @param SMWDataItem|null $value
+	 * @param SMWRequestOptions|null $requestOptions
 	 *
-	 * @return array of SMWDIWikiPage
+	 * @return array of DIWikiPage
 	 */
-	public function getPropertySubjects( SMWDIProperty $property, $value, $requestoptions = null ) {
+	public function getPropertySubjects( SMWDIProperty $property, SMWDataItem $value, SMWRequestOptions $requestOptions = null ) {
```

Comment:
Can be null `SMWDataItem|null $value`


PHP Catchable fatal error:  Argument 2 passed to SMWSQLStore3Readers::getPropertySubjects() must be an instance of SMWDataItem, null given, called in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/storage/SQLStore/SMW_SQLStore3_Readers.php on line 537 and defined in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/storage/SQLStore/SMW_SQLStore3_Readers.php on line 409